registry/idot_context.jsonld

```diff
@@ -261,7 +271,7 @@
     "MESH.2012": "http://identifiers.org/mesh.2012/",
```

Comment:
is the 2012 bit really required for our purposes? Are we in a mixed version environment?

Reply:
This is a direct transform of the idot registry. The role of this repo is to be a clearinghouse, from where intelligent decisions about what to mix and match can be made